Orange/tests/test_table.py

```diff
@@ -1254,7 +1254,7 @@ def test_is_sparse(self):
         self.assertTrue(table.is_sparse())
 
     def test_repr_sparse_with_one_row(self):
-        table = data.Table("iris")[:1]
+        table = data.Table("iris")[::50]
```

Comment:
Why this change? If we change this, then the name `test_repr_sparse_with_one_row` does not fit anymore since we have three instances.